### PR TITLE
✨amp-analytics: Add support for TEA Analytics

### PR DESCRIPTION
**Pull request by Harx**
related issue #18887

### CODE REVIEW

**Comment by Harx** [comment]
I signed it!

**Comment by Harx** [comment]
@zhouyx Could you please review code for me? 😃

**Comment by zhouyx** [comment]
@Harx Thank you for the Pull Request! I am currently on a trip and don't want to block you. @torch2424 could you please review this PR? Thank you!

File: examples/analytics-vendors.amp.html [hunk 1]
```diff
@@ -1289,6 +1325,7 @@
   </script>
 </amp-analytics>
 <!-- End Umeng Analytics example -->
+
```
Comment (torch2424) [inline examples/analytics-vendors.amp.html]: nit: Can we remove this newline?

**Comment by torch2424** [comment]
Thanks for pinging me by the way @zhouyx ! 😄

File: examples/analytics-vendors.amp.html (hunk 1 above)
Comment (Harx) [reply examples/analytics-vendors.amp.html]: > nit: Can we remove this newline?

done~

**Comment by Harx** [comment]
@torch2424 Thanks for review 😁！

**Pull request merged**